.github/workflows/integration.yaml

```diff
@@ -0,0 +1,107 @@
+name: 'Integration'
+
+on:
+  push:
+    branches:
+    - 'main'
+  pull_request:
+    branches:
+    - 'main'
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+permissions:
+  contents: 'read'
+  id-token: 'write'
+
+jobs:
+  test:
+    if: ${{ github.event_name == 'push' || github.repository == github.event.pull_request.head.repo.full_name && github.actor != 'dependabot[bot]' }}
+    runs-on: 'ubuntu-latest'
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: 'image'
+          image: 'gcr.io/cloudrun/hello:latest'
+
+    name: 'from_${{ matrix.name }}'
```

Comment:
Are there multiple ways to create a cloud deploy release? We should only use a matrix when there are multiple items in the matrix.

Reply:
There's scope for a future test variant, but I've removed this for now as it's not currently needed.